packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx

```diff
@@ -23,27 +23,36 @@ export type TableFieldProps<TData extends object> = {
 };
 
 export const TableField = <TData extends object>({ label, data, onChange, columns, emptyDataObject }: TableFieldProps<TData>) => {
+  const [tableData, setTableData] = useState(data);
+  const changeData = useCallback(
+    (change: TData[]) => {
+      setTableData(change);
+      onChange(change.filter(obj => !deepEqual(obj, emptyDataObject)));
+    },
+    [emptyDataObject, onChange]
+  );
```

Comment:
fix do not send empty rows to backend by:
- hold table data in own state
- filter empty row from onChange data, but not from internal state